website/content/api-docs/secret/gcp.mdx

```diff
@@ -505,6 +636,13 @@ $ curl \
     https://127.0.0.1:8200/v1/gcp/roleset/my-token-roleset/token
 ```
 
+```shell-session
+$ curl \
+    --header "X-Vault-Token: ..." \
+    --request GET \
+    https://127.0.0.1:8200/v1/gcp/impersonated-account/my-token-impersonate/token
+```
```

Comment:
Might be nice to include token requests for all three types? 

Something like:
```suggestion
**Roleset:**
    ```shell-session
$ curl \
    --header "X-Vault-Token: ..." \
    --request GET \
    https://127.0.0.1:8200/v1/gcp/roleset/my-token-roleset/token
    ```

**Static account:**
    ```shell-session
$ curl \
    --header "X-Vault-Token: ..." \
    --request GET \
    https://127.0.0.1:8200/v1/gcp/static-account/my-token-roleset/token
    ```

**Impersonated account:**
    ```shell-session
$ curl \
    --header "X-Vault-Token: ..." \
    --request GET \
    https://127.0.0.1:8200/v1/gcp/impersonated-account/my-token-impersonate/token
    ```
```

Reply:
Sorry, formatting got messed up a bit on the suggestion.. backticks in backticks.

Reply:
Yeah no reason to leave 1 out. Added in my commit due to the formatting.